hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/traceability/TraceabilitySuite.java

```diff
@@ -1697,7 +1696,7 @@ final HapiSpec traceabilityE2EScenario4() {
     @HapiTest
     @Order(6)
     final HapiSpec traceabilityE2EScenario5() {
-        return defaultHapiSpec("traceabilityE2EScenario5", NONDETERMINISTIC_FUNCTION_PARAMETERS)
+        return defaultHapiSpec("traceabilityE2EScenario5", FULLY_NONDETERMINISTIC)
```

Comment:
Again, for my own information, why are these tests suddenly needing `FULLY_NONDETERMINISTIC` over `NONDETERMINISTIC_FUNCTION_PARAMETERS`?   Since only a couple of them do, the rest are happy with `NONDETERMINISTIC_FUNCTION_PARAMETERS` and `NONDETERMINISTIC_TRANSACTION_FEES` together.

Reply:
Answered above